app/src/test/groovy/pl/futurecollars/invoicing/services/FileServiceTest.groovy

```
@@ -0,0 +1,19 @@
+package pl.futurecollars.invoicing.services
+
+import spock.lang.Specification
+
+class FileServiceTest extends Specification {
+
+    private FileService fileService
+
+    def "should create file"() {
+        given:
+        def path = "app/src/test/resources/inv.json"
+
+        when:
+        fileService.readInvoicesFromFile(path)
+
+        then:
+        thrown(NullPointerException)
```

test 'should create file' so why it throws null pointer exception?

---

I have changed name of test method

---

but why it should throw an exception? please add more details to test e.g:

"should throw exception if database file doesn't exist"

---

have changed name of test to "should read not existing file and throw exception"